driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/ODBFeatures.java

```diff
@@ -107,7 +108,7 @@ public boolean supportsComputer() {
 
         @Override
         public boolean supportsTransactions() {
-            return false;
+            return transactionalGraph;
```

Comment:
I reverted my change due to test failures. Happy to keep the discussion alive. What's your thoughts @velo ?